src/types/Variant.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Variant {
+  id: number;
+  assessmentId: number;
+}
```

Comment:
maybe combine related types into single file for overview and easy update

Reply:
Do we have a policy on this? I got expressly told by (I think) @sidohaakma that we put them in separate files for ehm... overview and easy update? :)